dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/Properties/launchSettings.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:61594",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    "http": {
```

Comment:
We should only have HTTPS here. Please remove all the others.

Reply:
Sorted.